feat: Adding a configurable additional ssh port. (#2838)
Azure/aks-engine: docs/topics/clusterdefinitions.md

| customVMTags                                                   | no                                                                                                | Specifies a list of custom tags to be added to the master VMs or Scale Sets. Each tag is a key/value pair (ie: `"myTagKey": "myTagValue"`).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |

> | sshAlternativePort                                              | no                                                                                                | If set, it would enable an alternative port for ssh connection in case that the original port 22 is banned for some reason.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |

Reviewer: ```suggestion
| sshAlternativePort                                              | no                                                                                                | Specifies an alternative port for ssh connection to be enabled in addition to the default port 22.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
```
